Providers/Modules/NPM/Plugin/plugin/plugin.rb

```diff
@@ -0,0 +1 @@
+this is a 32-bit binary placeholder
```

Comment:
are these binary files as well? .conf and .rb should not be binary. Not sure if this is github quirk?